### ISSUE TITLE
C20 Functional Bugs

### ISSUE THREAD

**Issue by Zerrisx** [magefree/mage#6492]
- [ ] AI didn't lose life when activating Netherborn Altar (although commander was not in command zone)
![image](https://user-images.githubusercontent.com/59861365/80638061-7fc10280-8a25-11ea-89ac-f134fdad82ae.png)

- [ ] Flawless Maneuver generates a rollback error when cast without a commander out
![image](https://user-images.githubusercontent.com/59861365/80637987-64ee8e00-8a25-11ea-8120-5ed25cc4e9ad.png)

- [ ] Boneyard Mycodrax is counting itself, which it shouldn't
![image](https://user-images.githubusercontent.com/59861365/80639354-80f32f00-8a27-11ea-8b73-5404595e5ab6.png)

**Comment by Zerrisx** [comment]
- [ ] In the event that Slippery Bogbonder enters with counter(s) on it (in this case, from Tayam), you don't seem to be able to move them as part of its ETB.
![image](https://user-images.githubusercontent.com/59861365/80660630-e22ff800-8a51-11ea-8ee3-cba7a5ea76df.png)

- [ ] If you choose a creature to move counters with for Slippery Bogbonder, but then say you want to move zero counters, it throws a rollback error.
![image](https://user-images.githubusercontent.com/59861365/80660754-333fec00-8a52-11ea-8adc-6607899b3de2.png)

**Comment by luziferius** [comment]
>     Akim, the Soaring Wind has a weird line about whether you've made a token yet this turn - and it has it all the time, _even if you have_.  I don't think it should be there at all ever, though.

This looks like a dynamical reminder text that tells the state of the card. The oracle text reads “Whenever you create one or more tokens _for the first time each turn_, […]”. This line tells, if it is still possible to trigger. It should show a green symbol, if no tokens were created previously in the turn to indicate whether this ability can trigger. It should change to red, if it already happened that turn and therefore can’t happen again this turn.

**Comment by JayDi85** [comment]
It's uses same condition for card's hint. So if you see wrong remind/hint text then condition and card's code is bugged too.

![shot_200430_141538](https://user-images.githubusercontent.com/8344157/80699479-14426800-8aed-11ea-8028-493caf605545.png)

**Comment by Zerrisx** [comment]
> 
> 
> It's uses same condition for card's hint. So if you see wrong remind/hint text then condition and card's code are bugged too.
> 
> ![shot_200430_141538](https://user-images.githubusercontent.com/8344157/80699479-14426800-8aed-11ea-8028-493caf605545.png)

That's the odd thing - functionally, the card is *correct*.  It properly tracks tokens made per turn, even if they happened before the card was cast or on opponent's turns.  But the dynamical reminder text never changes.

Also, either this card shouldn't have reminder text, or a lot more cards should.  Just from C20/IKO:

- [ ] Yidaro, Wandering Monster

- [ ] Gavi, Nest Warden

- [ ] Kalamax, the Stormsire

- [ ] Rielle, the Everwise

- [ ] Spellpyre Phoenix

- [ ] Titan Hunter

- [ ] Valiant Rescuer

Where I'd argue that Yidaro probably *should* (it's an annoying memory issue across multiple turns), but anything within the same turn is simple enough for the player to remember to check the chat log for.

**Comment by JayDi85** [comment]
So card hint must be simple inverted (e.g. green on no token used and red after first token trigger)

**Comment by Zerrisx** [comment]
I think I'd argue for just stripping the dynamical reminder text on this card entirely, rather that correcting it.  It's not hard to check the chat log to see if you've made a token yet within a single turn, and whenever possible we should remove visual clutter.

**Comment by JayDi85** [comment]
> whenever possible we should remove visual clutter

Nope, it's bad. The main ideology behind card's hints: card must contain all information for player decision, without manually calculations or searches in battlefield/logs.

**Comment by Zerrisx** [comment]
> 
> 
> > whenever possible we should remove visual clutter
> 
> Nope, it's bad. The main ideology behind card's hints: card must contain all information for player decision, without manually calculations or searches in battlefield/logs.

If that's the paradigm, I can create a list of all the cards that *should* have one, since it's a relatively long list.

**Comment by JayDi85** [comment]
It's will be good. Card hints are new for xmage (introduced in last year), so many cards or abilities can be improved with different info and hints.

**Comment by Zerrisx** [comment]
> 
> 
> It's will be good. Card hints are new for xmage (introduced in last year), so many cards or abilities can be improved with different info and hints.

Can do!

**Comment by theelk801** [comment]
Yeah I can definitely help add hints to things, I keep forgetting that they're available. I'll add one to Spellpyre Phoenix when I fix it.
